files/image_config/config-setup/config-setup

```diff
@@ -281,15 +380,51 @@ copy_post_migration_hooks()
     fi
 }
 
+# Get the list of config db for both
+# single and multi-npu platforms
+get_config_db_file_list()
+{
+    config_db_file_list=${CONFIG_DB_PREFIX}${CONFIG_DB_SUFFIX}
+    asic_num=0
+    while [[ ($asic_num -lt $NUM_ASIC) && ($NUM_ASIC -gt 1) ]]
+    do
+        config_db_file_list+=' '${CONFIG_DB_PREFIX}$asic_num${CONFIG_DB_SUFFIX}
+        ((asic_num = asic_num + 1))
+    done
+
+    echo $config_db_file_list
+}
+# Check if all needed config db are prsesnt for both
+# single and multi-npu platforms
+check_config_db_present()
```

Comment:
suggest rename this function to check_all_config_db_present() to match its return values.

Reply:
@yxieca Done